diff --git a/config/gmakegentest.py b/config/gmakegentest.py
index b998f31aa1..2f8a6ef44e 100755
--- a/config/gmakegentest.py
+++ b/config/gmakegentest.py
@@ -376,7 +376,7 @@ class generateExamples(Petsc):
     subst['rm']=self.conf['RM']
     subst['grep']=self.conf['GREP']
     subst['petsc_lib_dir']=self.conf['PETSC_LIB_DIR']
-    subst['wpetsc_dir']=self.conf['wPETSC_DIR']
+    subst['wpetsc_dir']=self.conf.get('wPETSC_DIR', '')
 
     # Output file is special because of subtests override
     defroot = testparse.getDefaultOutputFileRoot(testname)
